view/infinite.go

```diff
@@ -0,0 +1,431 @@
+package view
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jsdelivr/globalping-cli/client"
+	"github.com/jsdelivr/globalping-cli/model"
+	"github.com/mattn/go-runewidth"
+	"github.com/pterm/pterm"
+)
+
+// Table defaults
+var (
+	colSeparator = " | "
+)
+
+func OutputInfinite(id string, ctx *model.Context) error {
+	fetcher := client.NewMeasurementsFetcher(client.ApiUrl)
+	res, err := fetcher.GetMeasurement(id)
+	if err != nil {
+		return err
+	}
+	// Probe may not have started yet
+	for len(res.Results) == 0 {
+		time.Sleep(ctx.APIMinInterval)
+		res, err = fetcher.GetMeasurement(id)
+		if err != nil {
+			return err
+		}
+	}
+
+	if ctx.Latency || ctx.JsonOutput {
```

Comment:
@jimaek wdyt should happen with `--infinite --latency?` Right now, it just keeps printing individual measurements in the latency format, which I don't think is useful.

I previously suggested not supporting it at all and showing an error, but now I think we could just use `--latency` to force the table mode (when combined with `--infinite`). So with `limit` > 1 it would have no effect (table mode already forced), and with `limit` == 1 it would use the table as well since it's still "brief enough" but can keep updating in one line instead of printing new lines.

Reply:
Yeah updating the line sounds good to me